schema/ChallengeSchema.js

```
@@ -39,7 +40,10 @@ const Challenge = mongoose.model('Challenge', challengeSchema)
 
 
 module.exports = {
-  addChallenge: (data) => new Challenge(data).save(),
+  addChallenge: (data) => {
+    data.description = sanitize(data.description);
+    return new Challenge(data).save();
+  },
```

Imo the api should throw an error when some bad word is found

---

done